just_audio_background/lib/just_audio_background.dart

```diff
@@ -285,41 +284,35 @@ class _JustAudioPlayer extends AudioPlayerPlatform {
       _playerAudioHandler.customSetShuffleOrder(request);
```

Comment:
And this future should be returned.

Reply:
You may be viewing old files try to show changes from all commits:
![image](https://user-images.githubusercontent.com/8970959/204177383-66e56eb2-f11e-4a08-a9b8-c6743cc12876.png)

Reply:
It's actually returned as you can see here:

https://github.com/ryanheise/just_audio/pull/880/files/1368d8b67104865cfe13e5bc31c43b6f8dbba600#diff-c2bbf6ff0f0e12197c04baff3b8347fc40a1293d426d9f46016ce848a621b15fL282-L285